mk/tests.mk

```diff
@@ -650,7 +650,8 @@ CTEST_COMMON_ARGS$(1)-T-$(2)-H-$(3) := \
 	--python $$(CFG_PYTHON) \
         --gdb-version="$(CFG_GDB_VERSION)" \
         --lldb-version="$(CFG_LLDB_VERSION)" \
-        --android-cross-path=$(CFG_ANDROID_CROSS_PATH) \
+        --aarch64-linux-android-ndk=$(CFG_AARCH64_LINUX_ANDROID_NDK) \
+        --arm-linux-androideabi-ndk=$(CFG_ARM_LINUX_ANDROIDEABI_NDK) \
```

Comment:
This can probably be left as --android-cross path and these lines unchanged. compiletest is never going to be used to test against two android ndks simultaneously.